src/data-tutorials/users.tsx

```diff
@@ -789,6 +789,13 @@ const Users: User[] = [
     website: 'blog/tutorials/React-Native-&-Celo-Easily-build-React-Native-dApps-on-Celo',
     tags: ['intermediate','foundation', 'reactnative', 'composer'],
   },
+  {
+    title: 'Creating a Smart Contract for Decentralized Voting',
+    description: 'learn how to build a decentralised voting application using smart contracts on the Celo blockchain',
+    preview: require('./showcase/intermediate/Creating-a-Smart-Contract-for-Decentralized-Voting.png'),
```

Comment:
The name of the image reference on preview has the be the exact same name of the image you uploaded

Reply:
It's not the name of the image you uploaded in the intermediate folder